test/yamlSchemaService.test.ts

```diff
@@ -67,6 +67,50 @@ describe('YAML Schema Service', () => {
       expect(schema.schema.type).eqls('array');
     });
 
+    it('should handle schemas that use draft-04', async () => {
+      const content = `openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Minimal ping API server
+paths:
+  /ping:
+    get:
+      responses:
+        '200':
+          description: pet response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Pong'
+components:
+  schemas:
+    # base types
+    Pong:
+      type: object
+      required:
+        - ping
+      properties:
+        ping:
+          type: string
+          example: pong`;
+
+      const yamlDock = parse(content);
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const openapiV3Schema = require(path.join(__dirname, './fixtures/sample-openapiv3.0.0-schema.json'));
+
+      requestServiceMock = sandbox.fake.resolves(JSON.stringify(openapiV3Schema));
```

Comment:
Felt this is a bit redundant but not sure what's the best way to do it. require parses the json but the requestService response must be a string so I stringify it. Later it gets parsed again into an object by the code that loads the schema.

Reply:
I fixed this by loading the file using `readFile` instead of require.